src/components/common/Chat/index.js

```diff
@@ -21,6 +21,7 @@ const Chat = () => {
   }, [dispatch, t]);
 
   const itemTemplate = match => {
+    console.log(match);
     return (
```

Comment:
```suggestion
  const itemTemplate = match => (
```
You can implicitly return